### DISCUSSION TITLE
关于翻译后的结果复制

### DISCUSSION THREAD

**Discussion by yangiit** [YiiGuxing/TranslationPlugin#6198] [Ideas]
![2](https://github.com/user-attachments/assets/6ee1b8b4-fd68-45bc-b210-25fd2cfcf007)
作者你好，请问下面这跨区域可以增加选择和复制功能吗

**Comment by yangiit** [comment]
我用的是pycharm

**Comment by YiiGuxing** [comment]
这个比较麻烦，我后面再考虑考虑。
